package/MDAnalysis/analysis/leaflet.py

```diff
@@ -79,6 +79,7 @@
 
 from .. import core
 from . import distances
+from .. import selections
```

Comment:
Wow, really goes to show that tests are useful :) 

If you want, you could technically raise an issue about this and fix it within this PR, but either way, this should be recorded in the changelog.

Reply:
Thanks for your comments! I have done accordingly:)